src/OpenTelemetry/Metrics/MetricStreamConfiguration.cs

```diff
@@ -91,11 +95,39 @@ public class MetricStreamConfiguration
         }
     }
 
+    /// <summary>
+    /// Gets or sets a positive integer value
+    /// defining the maximum number of data points allowed to
+    /// per view.
+    /// </summary>
+    /// <remarks>
+    /// Note: If there is no matching view, but the MetricReader
+    /// defines a default cardinality limit value based on the
+    /// instrument an aggregation is created for, that value
+    /// will be used. The default value of 2000 will be used
+    /// if neither the view nor the MetricReader configures
+    /// MaxMetricPointsPerMetricStream.
+    /// </remarks>
+#if EXPOSE_EXPERIMENTAL_FEATURES
+    public
```

Comment:
@Yun-Ting As a follow-up PR let's add a diagnostic here. Something like this...

```csharp
#if NET8_0_OR_GREATER
    [Experimental(DiagnosticDefinitions.CardinalityLimitExperimentalApi, UrlFormat = DiagnosticDefinitions.ExperimentalApiUrlFormat)]
#endif
```

You can use this PR as a guide for how to do it: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5153

Reply:
Ended up using the same PR for adding the OTEL1003 definition/decoration.  :grin:
Hope this is not too nasty to review.